battstat/power-management.c

```diff
@@ -14,7 +14,7 @@
  *
  *  You should have received a copy of the GNU General Public License
  *  along with this program; if not, write to the Free Software
- *  Foundation, Inc., 59 Temple Street #330, Boston, MA 02110-1301, USA.
+ *  Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
```

Comment:
`Free Software` already exists on the end of the previous line.

Reply:
Yes, seen after create PR, correct now, the others too.

Thanks